content/rancher/v2.x/en/operation/ingress/externalLB/_index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: External LB
+weight: 2300
+draft: true
+version: v2.0
+lang: en
+---
+
+---
+
+**Introduction**
+
+If you want to use external lb L7, (ALB, nginx, F5,...) to provide access to services running at k8s cluster, this would be the scenario.
```

Comment:
It looks like what is described here is:
ExternalLB -> K8s Ingress -> Service

I wonder what the purpose of having a second L7 proxy in the form of the ingress?
Isn't the more common scenario for using external LB, to create a `nodePort` service?

`ExternalLB` -> `NodePort Service`

Reply:
Yes, i got your point, but seems some people want to use it. 
May be using `nodeport` should be placed in another section.